app/src/main/java/com/github/pockethub/android/core/PageIterator.java

```diff
@@ -56,11 +57,11 @@ public void remove() {
 
     @Override
     public List<V> next() {
-        if(!this.hasNext()) {
+        if (!this.hasNext()) {
```

Comment:
I've done a couple of those, right now I rather fix the things when I'm close to them. No it's not exactly what the PR is intended to do but it keeps the code to the checkstyle.